packages/lesswrong/components/posts/PostsTimeBlock.tsx

```diff
@@ -188,7 +189,7 @@ const PostsTimeBlock = ({ terms, timeBlockLoadComplete, startDate, hideIfEmpty,
             }}
           />}
           
-          {timeframe==="daily" && <TagEditsTimeBlock
+          {timeframe==="daily" && includeTags && <TagEditsTimeBlock
```

Comment:
I added this check to prevent tags from showing up under `/pastEvents`, which they currently do.